### ISSUE TITLE
Active state not updated when the active window is minimized

### ISSUE THREAD

**Issue by Tamaranch** [WayfireWM/wayfire#1660]
**Describe the bug**
`ZWLR_FOREIGN_TOPLEVEL_HANDLE_V1_STATE_ACTIVATED` is not updated when the active window is minimized.

**To Reproduce**
1. Open a window and make it active manually if necessary.
2. Minimize/unminize it by some means (e.g. tasklist button).
3. Verify that the active state does not change (tracing…).

I didn't bother writing a minimal reproducer, I just traced the problem locally. It should be easy enough to reproduce.

**Expected behavior**
`ZWLR_FOREIGN_TOPLEVEL_HANDLE_V1_STATE_ACTIVATED` should be updated.

**Wayfire version**
Wayfire git-master https://github.com/WayfireWM/wayfire/commit/0fb53ebc32311e59e5182f0a8d8ec11276d0e729.
Wayfire 0.7.4 worked correctly.

**Comment by ammen99** [comment]
I can easily believe it is broken on master, but are you sure it is broken on 0.7.5 ? The difference between 0.7.4 and 0.7.5 is very small (iirc one commit + the update to wlroots 0.16).

**Comment by Tamaranch** [comment]
Ah you're right sorry, I didn't see that there was a 0.7.x branch and I misidentified the date of the 0.7.5 tag with an approximate date on master. I'll update also the other issues that I recently opened.

Explanation: we don't have 0.7.5 on Arch Linux, only 0.7.4 or git, because of wlroots conflict, so I never actually used 0.7.5. Only 0.7.4 or a certain master commit (which I did well to specify :) )
